@keithjgrant There is one failing test
```
FAIL src/screens/Setting/SettingList.test.js
  ● Test suite failed to run

    Cannot find module '../../util/useBrandName' from 'src/screens/Setting/SettingList.test.js'

      4 |
      5 | jest.mock('../../api');
    > 6 | jest.mock('../../util/useBrandName', () => ({
        |      ^
      7 |   __esModule: true,
      8 |   default: () => ({
      9 |     current: 'AWX',

      at Resolver.resolveModule (node_modules/jest-runtime/node_modules/jest-resolve/build/index.js:306:11)
      at Object.<anonymous> (src/screens/Setting/SettingList.test.js:6:6)
```

--- a/awx/ui_next/src/screens/Setting/SettingList.test.js
+++ b/awx/ui_next/src/screens/Setting/SettingList.test.js
@@ -3,7 +3,7 @@ import { mountWithContexts } from '../../../testUtils/enzymeHelpers';
 import SettingList from './SettingList';
 
 jest.mock('../../api');
-jest.mock('../../util/useBrandName', () => ({
+jest.mock('hooks/useBrandName', () => ({
   __esModule: true,
   default: () => ({
     current: 'AWX',

